### ISSUE TITLE
SSRS Conversion: IEM4 - EnvironmentalImpactReport

### ISSUE THREAD

**Issue by HollySchuelerKCI** [kcigeospatial/sha_oed_issues#449]
### Additional Re-Design Specification: 
(None)

===================================================================
### Steps to Generate Report
1. Login to the EM Toolkit
2. Click the inspections tab, view subtab
3. Select tabular view radio button
![image](https://user-images.githubusercontent.com/25441512/45757134-ac2d5f80-bbf0-11e8-8711-3f143bd1d082.png)
4. Expand Search Panel
5. From the report dropdown, select impact report
![image](https://user-images.githubusercontent.com/25441512/45757212-d54df000-bbf0-11e8-879f-118e652ac043.png)
6. Click Search
7. From the search results, select the desired project
8. Scroll to the bottom, and click generate report
![image](https://user-images.githubusercontent.com/25441512/45757501-85235d80-bbf1-11e8-980c-c3dac33c4158.png)

===================================================================
### Example of Report
**Full Downloadable Report:** [IEM4 - EnvironmentalImpactReport.pdf](https://github.com/kcigeospatial/sha_oed_issues/files/2398425/IEM4.-.EnvironmentalImpactReport.pdf) 
 (First page below)

![image](https://user-images.githubusercontent.com/25441512/45757885-62de0f80-bbf2-11e8-82f7-b48a31ed85cf.png)

**Comment by sgpapakirk** [comment]
@HollySchuelerKCI, I have now finished coding this report.
First note that there is a second way to generate this report:
1) Log into the EM Toolkit.
2) From the inspections Tab, select a project from the list (pick one that you know it has at least one Environmental Impact report).
3) Click now on the top "Environmental Impacts" tab.
4) The "Environmental Impact List" page you should now see one or more records shown.
5) Pick one record from the grid, and click on its rightmost "print" icon.

Finally, I just merged my "Stylianos_Dev" code branch into the "master" and the "viji-reports" branches.

**Comment by vijme** [comment]
Deployed

**Comment by HollySchuelerKCI** [comment]
@sgpapakirk Please make one minor adjustment. Please reduce the font size by 1 or 2pts for the Disclaimer section.

**Comment by sgpapakirk** [comment]
@HollySchuelerKCI, I have now made the Disclaimer fields to be little smaller, 7.75pt.
I also did the same change in the https://github.com/kcigeospatial/sha_oed_issues/issues/449 ticket to be consistent.
Viji will deploy them both soon.

**Comment by HollySchuelerKCI** [comment]
This report has been successfully validated by the project team in the **_Development Environment._**

**Comment by Tad-Daniel** [comment]
Looks good

**Comment by HollySchuelerKCI** [comment]
This report has been successfully validated by the stakeholder in the **_Development Environment._**

**Comment by HollySchuelerKCI** [comment]
### FINAL REPORT OUTPUT SAMPLE:
[IEM 4 - EnvironmentalImpactReport - 449 - FINAL OUTPUT.pdf](https://github.com/kcigeospatial/sha_oed_issues/files/2725089/IEM.4.-.EnvironmentalImpactReport.-.449.-.FINAL.OUTPUT.pdf)

**Comment by sgpapakirk** [comment]
@HollySchuelerKCI, for this ticket I have implemented a change in the top header's green drop shadow. Previously if for some reason the shown fields would take more than 1 line, that would then push the rectangle's height, but it would have broken the looks of this custom green drop shadow.
I have implemented a new technique for it that allows vertical expansion. I researched and implemented it originally in the top header of the https://github.com/kcigeospatial/sha_oed_issues/issues/451 ticket, the "IEM5 - IssInsp" report.
